### PR TITLE
[DX-1781] Release notes for 5.7.1

### PR DESCRIPTION
**Pull request by Eopayemi**
### For internal users - Please add a Jira DX PR ticket to the subject!
[DX-1781](https://tyktech.atlassian.net/browse/DX-1781)

---

### Preview Link

<br>

### Description

Release notes for 5.7.1

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

[DX-1781]: https://tyktech.atlassian.net/browse/DX-1781?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Dashboard and Gateway version 5.7.1.
- Included sections for release highlights, breaking changes, dependencies, deprecations, upgrade instructions, and downloads.
- Introduced compatibility matrices for Tyk components and third-party dependencies.
- Added changelog sections for added, changed, and fixed items.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>dashboard.md</strong><dd><code>Added detailed release notes for Dashboard version 5.7.1.</code></dd></summary>
<hr>

tyk-docs/content/developer-support/release-notes/dashboard.md

<li>Added release notes for version 5.7.1, including highlights, breaking <br>changes, dependencies, deprecations, upgrade instructions, and <br>downloads.<br> <li> Introduced compatibility matrix for Tyk components and third-party <br>dependencies.<br> <li> Included changelog sections for added, changed, and fixed items.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5827/files#diff-d33730d9b7b352e9a2ce2f71ef67fea53fb21ae531d4522472f799648ee5b22f">+126/-0</a>&nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>gateway.md</strong><dd><code>Added detailed release notes for Gateway version 5.7.1.</code>&nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/developer-support/release-notes/gateway.md

<li>Added release notes for version 5.7.1, including highlights, breaking <br>changes, dependencies, deprecations, upgrade instructions, and <br>downloads.<br> <li> Introduced compatibility matrix for Tyk components and third-party <br>dependencies.<br> <li> Included changelog sections for added, changed, and fixed items.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5827/files#diff-d5455632656c7b0a876fad4078e8c28ade847c1ba3818941150f9b41738fa00a">+120/-0</a>&nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/content/developer-support/release-notes/dashboard.md [hunk 1]
```diff
@@ -37,6 +37,132 @@ Our minor releases are supported until our next minor comes out.
 
 ## 5.7 Release Notes
 
+### 5.7.1 Release Notes
+
+#### Release Date xxx
+
+#### Release Highlights
+
+This fix focuses mainly on bug fix.For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1" >}}) below.
```
Comment (sharadregoti) [inline tyk-docs/content/developer-support/release-notes/dashboard.md]:
```suggestion
This release contains a bug fix. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1" >}}) below.
```

File: tyk-docs/content/developer-support/release-notes/dashboard.md [hunk 2]
```diff
@@ -37,6 +37,132 @@ Our minor releases are supported until our next minor comes out.
...
+
+#### Breaking Changes
+<!-- Required. Use the following statement if there are no breaking changes, or explain if there are -->
```
Comment (sharadregoti) [inline tyk-docs/content/developer-support/release-notes/dashboard.md]:
```suggestion
```

Please remove all the commented stuff. [Condensed template](https://github.com/TykTechnologies/tyk-docs/blob/master/tyk-docs/assets/others/release-notes-condensed-template.md?plain=1) does not have any comments in it.

Do the same in gateway RN

File: tyk-docs/content/developer-support/release-notes/dashboard.md [hunk 3]
```diff
@@ -37,6 +37,132 @@ Our minor releases are supported until our next minor comes out.
...
+
+  
+##### Changed
+
+<ul>
+<li>
+<details>
+<summary>Changelog item summary</summary>
+
+Add changelog description
```
Comment (sharadregoti) [inline tyk-docs/content/developer-support/release-notes/dashboard.md]: Please update the change log with the bug fix & remove sections which are not required.

Do the same in gateway

File: tyk-docs/content/developer-support/release-notes/gateway.md [hunk 4]
```diff
@@ -44,6 +44,126 @@ Our minor releases are supported until our next minor comes out.
 
 ## 5.7 Release Notes
 
+### 5.7.1 Release Notes
+
+#### Release Date xxx
+
+#### Release Highlights
+
+This fix focuses mainly on bug fixes. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1">}}) below.
```
Comment (sharadregoti) [inline tyk-docs/content/developer-support/release-notes/gateway.md]:
```suggestion
This release contains a bug fix. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1">}}) below.
```

**Review by sharadregoti — CHANGES_REQUESTED** [review]

File: tyk-docs/content/developer-support/release-notes/dashboard.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/developer-support/release-notes/dashboard.md]: @sharadregoti should this be "....contains bug fixes."? (same for both files)

**Review by sharadregoti — APPROVED** [review]
TODO: Update release date while merging

File: tyk-docs/content/developer-support/release-notes/dashboard.md [hunk 5]
```diff
@@ -37,6 +37,78 @@ Our minor releases are supported until our next minor comes out.
 
 ## 5.7 Release Notes
 
+### 5.7.1 Release Notes
+
+#### Release Date xxx
+
+#### Release Highlights
+
+This release focuses mainly on bug fixes.For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1" >}}) below.
```
Comment (sharadregoti) [inline tyk-docs/content/developer-support/release-notes/dashboard.md]:
```suggestion
This release focuses mainly on bug fixes. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.7.1" >}}) below.
```

**Review by andyo-tyk — APPROVED** [review]

**Review by carolwkc — APPROVED** [review]

**Pull request merged**